packages/components/src/components/Select/Select.tsx

```diff
@@ -34,6 +34,11 @@ export type SelectComponentProps = {
      * The placeholder in the select component.
      */
     placeholder?: string;
+    /**
+     * Allow custom value in the combobox component.
+     * @default false
+     */
+    allowCustomValue?: boolean;
```

Comment:
Out of scope! Also Select doesn't have a text field, no custom values are possible

Reply:
for some reason, if I remove this I get a type error on the story. I added it as a temporary fix before I spent some time figuring out what's actually happening